pytorch_toolkit/ote/ote/modules/compression/nncf_config_generator.py

```diff
@@ -25,94 +25,118 @@
 
 # TODO(LeonidBeynenson): implement unit tests on NNCFConfigGenerator
 
-POSSIBLE_NNCF_PARTS = {'int8', 'sparsity', 'pruning'}
```

Comment:
Have you implemented the tests? See two lines above, if yes please remove TODO

Reply:
No, I have not implemented them yet.
So, the comment it still required